controllers/k8ssandra/medusa_reconciler.go

```diff
@@ -119,7 +120,7 @@ func (r *K8ssandraClusterReconciler) reconcileMedusa(
 		}
 		if !ready {
 			logger.Info("Medusa standalone deployment is not ready yet")
-			return result.RequeueSoon(r.DefaultDelay)
+			return result.Error(err)
```

Comment:
There's no err necessarily here, the method can return a `false, nil` in which case err is nil here.

Reply:
Great spot, fixed.